eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -619,4 +619,23 @@ public function publish( $contentTypeId )
 
         $this->updateHandler->publishNewType( $toType, Type::STATUS_DEFINED );
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
+     */
+    public function getFieldMap()
+    {
+        $fieldMap = [];
```

Comment:
> You mean Shippable?

I mean whatever runs them. Shippable or something else.

The fact is that our versions are gonna diverge more and more. It is still manageable, but it won't scale over time, meaning that we should think about improvements/bots-doing-the-work-for-us now, before it hits us by surprise :-)